front/src/applications/operationalStudies/views/Scenario/components/SimulationResults/SimulationResultsExport/SimulationResultsExport.tsx

```diff
@@ -49,8 +49,19 @@ const exportTrainPDF = async (
 
   const blob = await pdf(doc).toBlob();
   const url = URL.createObjectURL(blob);
-  window.open(url, '_blank');
-  URL.revokeObjectURL(url);
+
+  // Create a temporary link element for better browser compatibility
+  const link = document.createElement('a');
+  link.href = url;
+  link.target = '_blank';
```

Comment:
```suggestion
  link.target = '_blank';
  link.rel = 'noopener noreferrer';
```
seems to be a good practice for this kind of situation
https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/rel/noopener
https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/rel/noreferrer

Reply:
I added only `rel="noreferrer"` because according to its description, it already includes `noopener` behavior.